distinct_test.go

```diff
@@ -173,7 +181,6 @@ func TestDistinct(t *testing.T) {
 				Distinct(test.columns...).
 				Execute(context.Background(), func(ctx context.Context, ar arrow.Record) error {
 					defer ar.Release()
-					require.Equal(t, len(test.values), int(ar.NumRows()))
```

Comment:
Accidental deletion?

Reply:
No, this is intentional, since we sometimes get 1 row and then 2 rows or the other way around, using the callback concurrently under the hood. We just need to make sure we've seen all rows in the end.